data/core/units/dunefolk/Alchemist.cfg

```diff
@@ -18,9 +18,7 @@
     [abilities]
         {ABILITY_REGENERATES}
     [/abilities]
-    description= _ " Concept by Krogen
```

Comment:
This is in many other units description too, I just left one comment per class of issue.

Reply:
I suppose it doesn't matter, that are just the credits and won't be included in the game.